lib/IDE/CodeCompletion.cpp

```diff
@@ -2530,11 +2533,12 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
     addValueBaseName(Builder, Name);
     setClangDeclKeywords(VD, Pairs, Builder);
 
+    Optional<CodeCompletionResult::NotRecommendedReason> NotRecommended;
     // "not recommended" in its own getter.
     if (Kind == LookupKind::ValueInDeclContext) {
       if (auto accessor = dyn_cast<AccessorDecl>(CurrDeclContext)) {
         if (accessor->getStorage() == VD && accessor->isGetter())
-          Builder.setNotRecommended(CodeCompletionResult::NoReason);
+          NotRecommended = CodeCompletionResult::NoReason;
```

Comment:
We have an early-return below for `if (!VD->hasInterfaceType())`; I think this could change behaviour in that case.

Reply:
Thanks, I didn't notice that.